_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -28,6 +28,13 @@ Segment replication can be applied in a variety of scenarios, including:
 - When experiencing very high loads, you want to add new nodes but don't want to index all data immediately.
 - OpenSearch cluster deployments with low replica counts, such as those used for log analytics.
 
+## Remote store
+
+As of OpenSearch 2.10, segment replication now supports two methods for storing replica shards:
+
+- Using a **Remote Store**, a remote storage solution where you can store shards, your primary replica uploads segments to the remote store. Then, you can down the replica shards from that store. For more information about using a remote store, see [Remote backend storage]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/index/#segment-replication-and-remote-backed-storage)
```

Comment:
`you can store shards, your primary replica uploads segments to the remote store`

I think we can remove `you can store shards` from above phrase (and other places) as we not storing everything inside the index shards. For e.g. segment_N, write.lock etc. Also, with remote store we are not storing the shard metadata onto remote store. I think we should only state that with remote store, segments (data) files are stored on remote store. @sachinpkale please correct me if I am wrong.

Reply:
>  Then, you can down the

download ?